koku/api/report/test/__init__.py

```diff
@@ -1 +1,455 @@
-# noqa
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Unit testing utilities."""
+import logging
+import random
+import re
+from unittest.mock import Mock
+from urllib.parse import urlencode
+
+from faker import Faker
+
+from api.query_params import QueryParameters
+from api.utils import DateHelper
+
+LOG = logging.getLogger(__name__)
+
+
+class FakeAWSCostData:
```

Comment:
Moving this test class isn't required for the refactor. It was a "while I was there" decision to break apart the massive `test_queries` test file, which was approaching 2000 lines.